test/unittests.jl

```diff
@@ -1616,7 +1616,7 @@ end
         :coth, :asech, :acsch, :acoth, :sinc, :cosc, :cosd, :cotd, :cscd, :secd,
         :sinpi, :cospi, :sind, :tand, :acosd, :acotd, :acscd, :asecd, :asind,
         :log, :log2, :log10, :log1p, :exp, :exp2, :exp10, :expm1, :frexp, :exponent,
-        :atan, :atand
+        :atan, :atand, :factorial
```

Comment:
Probably not the right place for it as these test real input. You might just want to add a single test elsewhere on some input like `Quantity{Int64}(2u"1")`.